pkg/services/provisioning/provisioning.go

```diff
@@ -94,6 +96,7 @@ type ProvisioningServiceImpl struct {
 	provisionPlugins        func(context.Context, string, plugins.Store, plugifaces.Store) error
 	mutex                   sync.Mutex
 	dashboardService        dashboardservice.DashboardProvisioningService
+	dashAlertExtractor      alerting.DashAlertExtractor
```

Comment:
I see this dependency has been added into the provisioning service but it's not used, is that correct? 🤔

Reply:
Probably is a mistake :/